src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java

```diff
@@ -1,21 +1,32 @@
 package jenkins.plugins.office365connector.workflow;
 
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Set;
+import javax.annotation.Nonnull;
+
+import com.google.common.collect.ImmutableSet;
 import hudson.Extension;
+import hudson.Util;
 import hudson.model.Run;
 import hudson.model.TaskListener;
-import javax.inject.Inject;
+import hudson.util.FormValidation;
 import jenkins.plugins.office365connector.Office365ConnectorWebhookNotifier;
-import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
-import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
-import org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution;
-import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
 
 /**
  * Workflow step to send a notification to Jenkins office 365 connector.
  */
-public class Office365ConnectorSendStep extends AbstractStepImpl {
+public class Office365ConnectorSendStep extends Step {
```

Comment:
Workflow-step-api has a lower Jenkins Target then us, so completely safe rewrite
We use `1.651.2`
They use `1.642.3`
https://github.com/jenkinsci/workflow-step-api-plugin/blob/8b21acfb5b09ec25921b86b99d1202ec7b1e91ce/pom.xml#L65